sdk/src/utils/evm.rs

```diff
@@ -0,0 +1,88 @@
+use crate::events_watcher::Error;
+use alloy_network::EthereumWallet;
+use alloy_primitives::{Address, U256};
+use alloy_provider::{Provider, ProviderBuilder, RootProvider, WsConnect};
+use alloy_transport::BoxTransport;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    /// 1 day
+    pub static ref SIGNATURE_EXPIRY: U256 = U256::from(86400);
+}
```

Comment:
Remove `lazy_static` and use a constant? See: https://docs.rs/alloy/latest/alloy/primitives/struct.Uint.html#method.from_limbs

OR you can use https://doc.rust-lang.org/stable/std/cell/struct.LazyCell.html from the `std`.